api/read-text.md

```diff
@@ -25,6 +23,14 @@ GET
  * `uuid` : The UUID of the check to which the metric belongs.
  * `metric` : The name of the metric for which to pull data.
 
+### Output
+
+The response payload will contain a JSON array of tuples consisting of the timestamps (in millisecions since epoch) from the time the value was received, and the text value itself as a string.
+
+The first value of the response array is an exception to this rule.
```

Comment:
Instead of explaining this as an exception, put it at the top of this paragraph.   "The first value is the timestamp of the requesting parameter, and contains the value at the start of the time span.   All subsequent values contain the timestamp of the change, and the new value the metric changed to."  (or something like that) . This will read much clearer and be easier for someone to understand.